# An array of pose hypothesis 

Header header

# Camera's topic name
string topicName

geometry_msgs/Pose2D[] leftDot
geometry_msgs/Pose2D[] rightDot

# Focal length of the camera
float64 fx
float64 fy

# Center of the camera
float64 px
float64 py